src/FontLib/Table/Type/hmtx.php

```diff
@@ -35,8 +35,13 @@ protected function _parse() {
     }
 
     if ($numOfLongHorMetrics < $numGlyphs) {
-      $lastWidth = end($data);
-      $data      = array_pad($data, $numGlyphs, $lastWidth);
+      $lastWidth = end($data)[0];
+      $numLeft   = $numGlyphs - $numOfLongHorMetrics;
+      $metrics   = $font->readUInt16Many($numLeft);
+      for($i = 0; $i < $numLeft; $i++) {
+        $gid        = $numOfLongHorMetrics + $i;
+        $data[$gid] = array($lastWidth, $metrics[$i]);
```

Comment:
Should we do as above and set to 0 if, for unknown reasons, the left side bearing value is not set? Would an entry in the hmtx table ever not have a value?

Reply:
elements in `$data` array is set by line 34
```
$data[$gid]      = array($advanceWidth, $leftSideBearing);
```
so `$data[0]` always has value. It is "advance width".

Reply:
What I mean is that above we do `isset($metrics[$mid]) ? $metrics[$mid] : 0;` should we do the same here?

```
$leftSideBearing = isset($metrics[$i]) ? $metrics[$i] : 0;
$data[$gid] = array($lastWidth, $leftSideBearing);
```

Reply:
If the i-th element of the array does not exist, it indicates an error in the font file. Exception handling can be considered thereafter.

Reply:
Agreed. It seems unlikely that the metrics would be present but not set. I'll keep an eye out for potential issues there.